src/control/controlobject.cpp

```diff
@@ -94,6 +94,12 @@ double ControlObject::get(const ConfigKey& key) {
     return pCop ? pCop->get() : 0.0;
 }
 
+// static
+bool ControlObject::toBool(const ConfigKey& key) {
```

Comment:
- Why not inline?
- Shrink to a single line without parentheses

Reply:
Inline didn't work, it caused linker errors:

```
/usr/bin/ld: /usr/bin/ld: DWARF error: could not find variable specification at offset 84f37
/usr/bin/ld: DWARF error: could not find variable specification at offset 84f45
/usr/bin/ld: DWARF error: could not find variable specification at offset c3765
/usr/bin/ld: DWARF error: could not find variable specification at offset c396c
/usr/bin/ld: DWARF error: could not find variable specification at offset c39bd
libmixxx-lib.a(mixxx.cpp.o): in function `MixxxMainWindow::confirmExit()':
/home/jan/Projects/mixxx/src/mixxx.cpp:1607: undefined reference to `ControlObject::toBool(ConfigKey const&)'
/usr/bin/ld: /home/jan/Projects/mixxx/src/mixxx.cpp:1614: undefined reference to `ControlObject::toBool(ConfigKey const&)'
/usr/bin/ld: /usr/bin/ld: DWARF error: could not find variable specification at offset 84f37
/usr/bin/ld: DWARF error: could not find variable specification at offset 84f45
/usr/bin/ld: DWARF error: could not find variable specification at offset c3765
/usr/bin/ld: DWARF error: could not find variable specification at offset c396c
/usr/bin/ld: DWARF error: could not find variable specification at offset c39bd
libmixxx-lib.a(mixxx.cpp.o): in function `MixxxMainWindow::confirmExit()':
/home/jan/Projects/mixxx/src/mixxx.cpp:1607: undefined reference to `ControlObject::toBool(ConfigKey const&)'
/usr/bin/ld: /home/jan/Projects/mixxx/src/mixxx.cpp:1614: undefined reference to `ControlObject::toBool(ConfigKey const&)'
collect2: error: ld returned 1 exit status
```

Reply:
The other changes are done in: https://github.com/mixxxdj/mixxx/pull/3227/commits/3981e1bf7d45489f772789b8316fe260c4fc20fa

Reply:
Doh, I forgot. sorry. Fixed now.